label_studio_ml/examples/segment_anything_2_video/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.8"
+
+services:
+  ml-backend:
+    container_name: ml-backend
```

Comment:
@farioas please review what we should put here

Reply:
change made!